test/e2e/dra/dra.go

```diff
@@ -64,6 +66,15 @@ const (
 	podStartTimeout = 5 * time.Minute
 )
 
+var (
+	extendedResourceName  = "example.com/gpu"
```

Comment:
Let's replace with a function `extendedResourceName(index int)`. Then you can replace some code below with a for loop instead of using cut-and-paste.